kie-server-parent/kie-server-tests/kie-server-integ-tests-optaplanner/src/test/filtered-resources/kjars-sources/cloudbalance/src/main/java/org/kie/server/testing/CloudComputerStrengthComparator.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2010 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.server.testing;
+
+import java.io.Serializable;
+import java.util.Comparator;
+
+import org.apache.commons.lang3.builder.CompareToBuilder;
+
+public class CloudComputerStrengthComparator implements Comparator<CloudComputer>, Serializable {
```

Comment:
Overkill, I 'll send a PR to remove all code in the copy of the example that's overkill.